server/config.ts

```diff
@@ -93,10 +93,10 @@ export default {
     approvedPremises: {
       url: get('APPROVED_PREMISES_API_URL', 'http://localhost:9092', requiredInProduction),
       timeout: {
-        response: 10000,
-        deadline: 10000,
+        response: Number(get('COMMUNITY_ACCOMMODATION_API_TIMEOUT_RESPONSE', 10000)),
+        deadline: Number(get('COMMUNITY_ACCOMMODATION_API_TIMEOUT_RESPONSE', 10000)),
```

Comment:
I'd be tempted to use COMMUNITY_ACCOMMODATION_API_TIMEOUT_*DEADLINE*, to match TOKEN_VERIFICATION_API_TIMEOUT_*  and HMPPS_AUTH_TIMEOUT_*